modules/op-release-notes-1-19-4.adoc

```diff
@@ -0,0 +1,82 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-19.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-19-4_{context}"]
+= Release notes for {pipelines-title} 1.19.4
+
+With this update, {pipelines-title} General Availability (GA) 1.19.4 is available on {OCP} 4.16 and later versions.
+
+[id="fixed-issues-1-19-4_{context}"]
+== Fixed issues
+
+.Pipelines
+
+Git resolver no longer ignores certificates in the cluster custom PKI::
+Before this update, a regression caused the Git resolver to no longer use the OpenShift Proxy custom-configured PKIs. This prevented the Git resolver from resolving references to a self-hosted Git provider. With this update, the full CA bundle configured in the OpenShift Proxy is trusted by the system in all components, including the Git resolver.
++
+link:https://issues.redhat.com/browse/SRVKP-9177[SRVKP-9177]
+
+Operator no longer panics when validating v1beta1 Pipelines with invalid matrix variable substitutions::
+Before this update, a `Pipeline` resource using the v1beta1 API that included a matrixed task with an invalid variable or result reference caused the Operator validation webhook to panic. This occurred because the v1beta1 validation logic attempted to resolve malformed result substitutions inside `matrix` parameters, leading to an out-of-range index error and a webhook crash. As a result, `Pipeline` creation failed with an internal error instead of returning a proper validation error. With this update, the validation logic correctly detects invalid result references and rejects `Pipeline` resources with a standard validation error, preventing the Operator from panicking.
```

Comment:
Again, I am not in favor of the usage of the words "to panic" or "from panicking". It does not sound correct to me. Please change it to something else.